app/common/src/services/Backend.ts

```diff
@@ -1433,13 +1433,23 @@ export function compareAssets(a: AnyAsset, b: AnyAsset) {
 
   if (relativeTypeOrder !== 0) {
     return relativeTypeOrder
+  } else {
+    // We sort by modified date, because the running/recent projects should be at the top.
+    // but above the all folders
```

Comment:
Lets not start sentences with `but` and first word should be capitalised. Running projects should also appear first on the list but bellow catalogs